colossalai/auto_parallel/tensor_shard/sharding_strategy.py

```diff
@@ -115,6 +115,7 @@ class CommAction:
     comm_spec: CommSpec = None
     comm_type: CommType = None
     arg_index: int = -1
+    key_for_kwarg: any = None
```

Comment:
what is this used for?

Reply:
If a node is kwarg of other node, not a positional argument, we could use this to locate it in runtime pass.